### ISSUE TITLE
Origin request hangs forever

### ISSUE THREAD

**Issue by chandywerks** [evolv-ai/serverless-offline-edge-lambda#128]
I'm using serverless 2.11.1 and I'm having a hard time getting this plugin to work. In my origin request handler I am dynamically setting the origin but when I try it I get...

```
Serverless: Request for /    
Serverless: → viewer-request 
Serverless: → cache           
Serverless: ✗ Cache miss                                                                                                                 
Serverless: → origin-request
Serverless: → origin
```

And then the request just hangs forever. I've tried both https origins and S3 origins.

**Comment by l1b3r** [comment]
Hanging origin happens to me when there's no local file to "serve" from that origin.

Here's how I configure my origin and it works for me:

```yml
// serverless.yml
custom:
    offlineEdgeLambda:
        path: .build
        originMap:
            - pathPattern: '*'
              target: ./dev/origins
```

And then I have a subdirectory in my project that emulates some dummy origin responses:
```
dev
│
└───origins
    │   favicon.ico
    │   index
    ├─── . . .
```
Where `index` is a dummy JSON file: 
```json
{
    "message": "Hello index"
}
```

And finally if I run `sls` like this:
<pre>
sls offline start --port 8888 <strong>--fileDir ./dev/origins</strong> --disableCache true
</pre>
I can reach my `Hello index` message at `localhost:8888/index`.

I don't know how to attach a dummy file to a root (`locahost:8888/`) though...

**Comment by ryanolee** [comment]
This looks to be down to Boom not being handled properly? 
Given `Boom.notFound` is actually not properly defined in defined in `https://github.com/evolv-ai/serverless-offline-edge-lambda/blob/023470fc40bf331e0a21f12bd7d32dff1a385564/src/services/origin.service.ts#L49` causing the request to hang forever given the catch also errors out. Odd behaviour :thinking: 

In either case will open a PR to fix this shortly :thinking:

**Comment by mattstrom** [comment]
Resolved by #193
